### ISSUE TITLE
append the account name to the customization pipeline name

### ISSUE THREAD

**Issue by v-rosa** [aws-ia/terraform-aws-control_tower_account_factory#269]
Appending the account name to the customization pipeline name.

e.g. `1234567890-customizations-pipeline` to `1234567890-customizations-pipeline-awsome-account-name`

Having the account number its already an unique way to distinguish the pipelines, but for us, humans, it quite hard to memorize and map the account numbers to the account names.

**Comment by smokentar** [comment]
In my opinion this should be handled with tags - ref. #253 
Appending the account name to the workspace would create clutter

If #253 is implemented, we can add a `name` tag to the workspace to easily distinguish them

**Comment by v-rosa** [comment]
I'm not referring to TF workspaces, but to AWS Code Pipeline.

**Comment by hanafya** [comment]
Hey @v-rosa!

Thank you for this feature request! I have created a backlog item for this and will review with the team.

**Comment by jdebbink** [comment]
This would be a very useful addition. It is much easier to select a pipeline by account name rather than its ID.

**Comment by laurenty** [comment]
I agree that this would handled better with tags on the pipeline. It also would be great if AWS added the option to show tags on the pipelines list.

**Comment by Kirizan** [comment]
I concur that the name of the pipeline should reflect the name of the account. Even with tags, that doesn't help when looking for which of the CodeBuild run's we are looking for. Naming the pipeline after the account would also mean the submitter listed in CodeBuild would be easier to identify which run we are looking for when troubleshooting. I'm not sure about the name in the OP, but I'd go something like `aft-<accn number>-<accn name>`, or along those lines. By replacing the `customizations-pipeline`, you could keep the name from getting cluttered as smokentar mentioned.

**Comment by mlynch1985** [comment]
For those who don't want the clutter within the console, could we make this a opt-out feature?

**Comment by sputmayer** [comment]
How hard cant it be to sit in the backlog for so long? Having an option to add account names to the pipelines would be pretty helpful when searching for specific pipeline
